tap_mambu/helpers/schemas/activities.json

```diff
@@ -1,239 +1,85 @@
+                                "type": ["null", "string"]
+                            },
+                            "new_value": {
+                                "type": ["null", "string"]
+                            },
+                            "clientKey": {
+                                "type": ["null", "string"]
+                            },
+                            "branchKey": {
+                                "type": ["null", "string"]
+                            },
+                            "loanProductKey": {
+                                "type": ["null", "string"]
+                            },
+                            "loanAccountKey": {
+                                "type": ["null", "string"]
+                            }
+                        }
+                    }
+                },
+                {
+                    "type": "null"
+                }
+            ]
         }
-      ]
-    },
-    "gl_accounts_closure_key": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "gl_account_key": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "group_key": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "group_name": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "line_of_credit_key": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "loan_account_key": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "loan_account_name": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "loan_product_key": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "loan_product_name": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "notes": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "savings_account_key": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "savings_account_name": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "savings_product_key": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "savings_product_name": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "task_key": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "task_name": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "timestamp": {
-      "type": [
-        "null",
-        "string"
-      ],
-      "format": "date-time"
-    },
-    "transaction_id": {
-      "type": [
-        "null",
-        "integer"
-      ]
-    },
-    "type": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "user_key": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "user_name": {
-      "type": [
-        "null",
-        "string"
-      ]
     }
```

Comment:
There are a lot of fields removed in this file, and the fields in the underlying object are changed. Is this intentional? This is maybe not a hard breaking change, but will result in data not being populated for the fields that were removed.